packages/edit-site/src/components/dataviews/view-list.js

```diff
@@ -122,6 +136,102 @@ function HeaderMenu( { dataView, header } ) {
 					{ __( 'Hide' ) }
 				</DropdownMenuItemV2>
 			) }
+			{ ( isSortable || isHidable ) && isFilterable && (
+				<DropdownMenuSeparatorV2 />
+			) }
+			{ isFilterable && (
+				<DropdownMenuGroupV2>
+					{ hasSearchFilter && (
+						<DropdownMenuItemV2
+							key={ 'filter-search' }
+							prefix={ <Icon icon={ search } /> }
+							onSelect={ () => {
+								// TODO: implement search.
+							} }
+						>
+							{ __( 'Search' ) }
+						</DropdownMenuItemV2>
+					) }
+					{ hasInFilter && (
+						<DropdownSubMenuV2
+							trigger={
+								<DropdownSubMenuTriggerV2>
+									{ __( 'In' ) }
+								</DropdownSubMenuTriggerV2>
+							}
+						>
+							{ [
+								{ id: undefined, name: __( 'Any' ) },
+								...header.column.columnDef.setList,
+							].map( ( element ) => (
+								<DropdownMenuItemV2
+									key={ element.name }
```

Comment:
Super nit for this stage, but users can have the same name and we end up with duplicate keys.